### ISSUE TITLE
storybook/react@5.2.2 manual setup get Error "Can't resolve '@storybook/core/client'"

### ISSUE THREAD

**Issue by abrcdf1023** [storybookjs/storybook#8315]
**Describe the bug**
I followed the manual setup in [guide page](https://storybook.js.org/docs/guides/guide-react/) with empty project and got these errors.
```
ERROR in ./node_modules/@storybook/react/dist/client/preview/index.js
Module not found: Error: Can't resolve '@storybook/core/client' in '/Users/liyanxin/Life/myprojects/test/node_modules/@storybook/react/dist/client/preview'
 @ ./node_modules/@storybook/react/dist/client/preview/index.js 12:14-47
 @ ./node_modules/@storybook/react/dist/client/index.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true
```
**To Reproduce**
Please follow the manual setup to create a new project.

**Expected behavior**
It should not have any error.

**Screenshots**
<img width="1849" alt="螢幕快照 2019-10-07 18 48 29" src="https://user-images.githubusercontent.com/12894459/66305956-14053400-e933-11e9-9c25-486763ca19ff.png">

**Code snippets**
No

**System:**

Environment Info:

  System:
    OS: macOS Mojave 10.14.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  Binaries:
    Node: 10.15.0 - ~/.nvm/versions/node/v10.15.0/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.4.1 - ~/.nvm/versions/node/v10.15.0/bin/npm
  Browsers:
    Chrome: 77.0.3865.90
    Safari: 12.1.1
  npmPackages:
    @storybook/react: ^5.2.2 => 5.2.2 

**Additional context**
If I change to v5.1.10 it work.

**Comment by sbadulin** [comment]
Also have this problem after updating to 5.2.2

**Comment by selrond** [comment]
same issue as in #8314

**Comment by nrdlab** [comment]
Same here !!!  in 5.2.2

**Comment by selrond** [comment]
https://github.com/storybookjs/storybook/issues/8314#issuecomment-538958206

**Comment by sbadulin** [comment]
I've updated all dependencies with npx npm-check-updates -t -u '/storybook/' and now it's all good.

**Comment by nrdlab** [comment]
Fixed in `"5.2.3"` , just update your storybook and everythink will work as always !!!

**Comment by shilman** [comment]
Thanks guys. Sorry for the hassle!
